### ISSUE TITLE
Let's add the body to the UnexpectedResponseException

### ISSUE THREAD

**Issue by amihaiemil** [amihaiemil/docker-java-api#108]
Let's also add the body to the exception message, not just the expected/actual status

**Comment by 0crat** [comment]
Job [#108](https://github.com/amihaiemil/docker-java-api/issues/108) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
The job [#108](https://github.com/amihaiemil/docker-java-api/issues/108) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/G6LPQQV2P/1351434173); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html)

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting for merge of #111

**Comment by 0crat** [comment]
> @0crat waiting for merge of #111  ([here](https://github.com/amihaiemil/docker-java-api/issues/108#issuecomment-391391376))

@llorllale The impediment for [#108](https://github.com/amihaiemil/docker-java-api/issues/108) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by llorllale** [comment]
@amihaiemil can we close this? PR #111 was merged

**Comment by amihaiemil** [comment]
@llorllale As far as I can tell, PR #111 has nothing to do with this task :D

**Comment by llorllale** [comment]
@amihaiemil oops, my bad

**Comment by llorllale** [comment]
@0crat waiting on PR #126 for solving this

**Comment by amihaiemil** [comment]
@0crat boost x5

**Comment by 0crat** [comment]
> @0crat boost x5 ([here](https://github.com/amihaiemil/docker-java-api/issues/108#issuecomment-393454274))

@amihaiemil Argument "x5" doesn't match regex `\d+x`

**Comment by amihaiemil** [comment]
@0crat boost 5x

**Comment by 0crat** [comment]
> @0crat boost 5x ([here](https://github.com/amihaiemil/docker-java-api/issues/108#issuecomment-393454683))

@amihaiemil Boost 5x was set for [#108](https://github.com/amihaiemil/docker-java-api/issues/108)

**Comment by 0crat** [comment]
Boosting tasks is against our principles, see [§15](http://www.zerocracy.com/policy.html#15): -10 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Order was finished: +75 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
The job [#108](https://github.com/amihaiemil/docker-java-api/issues/108) is now out of scope
